script.js

```diff
@@ -0,0 +1,88 @@
+let defaultContainer = document.getElementById("defaultContainer")
+let divColor = document.getElementById("text")
+let dimension = document.getElementById("dimension")
+let container = document.getElementById("container")
+// functions for different kinds of colors
+function red(){
+    divColor.textContent = "RED"
+}
+function yellow(){
+    divColor.textContent = "YELLOW"
+}
+function green(){
+    divColor.textContent = "GREEN"
+}
+function blue(){
+    divColor.textContent = "BLUE"
+}
+function eraser(){
+    divColor.textContent = "ERASER"
+}
+
+for (let j = 1 ; j<=256 ; j++){ 
+    let defaultContent = document.createElement("div"[j])
+    defaultContainer.appendChild(defaultContent)
+    defaultContent.style.width = "25px" 
+    defaultContent.style.height = "25px"
+    defaultContent.addEventListener("mouseover", hover)
+    function hover(){
+        if(divColor.textContent === "BLUE"){
+            defaultContent.style.background = "blue" 
+        }
+      else if(divColor.textContent === "RED"){
+            defaultContent.style.background = "red" 
+        }
+        else if(divColor.textContent === "YELLOW"){
+            defaultContent.style.background = "yellow" 
+        }
+        else if(divColor.textContent === "GREEN"){
+            defaultContent.style.background = "green" 
+        }
+       else if(divColor.textContent === "ERASER"){
+            defaultContent.style.background = "WHITE" 
+        }
+        else{
+            defaultContent.style.background = "black" 
+        }
+    }
+}
+function edit(){
+    clear()
+    defaultContainer.style.display = "none"
+    let askDimension = Number(prompt("enter dimension"))
```

Comment:
What if the user enters something that's not a number? 😉

Reply:
OR EVEN A DIMENSION THAT'S TOO LARGE 😬
I can see issues everywhere now

Reply:
i have fixed this now,thanks alot!